src/components/Checkbox/docs/index.story.tsx

```diff
@@ -86,7 +101,7 @@ export default {
               params: {
                 colors: [
                   {
-                    label: 'Icon Color',
+                    label: 'Icon/Box Color',
```

Comment:
Icon and box colors aren't supposed to be the same.
We'll need the two separate colors to be customizable.

Reply:
Ah OK, I thought because they both use color-5 it should be the same var.